src/Mod/Sketcher/Gui/DrawSketchHandlerEllipse.h

```diff
@@ -261,7 +261,15 @@ class DrawSketchHandlerEllipse: public DrawSketchHandlerEllipseBase
 
     QString getCrosshairCursorSVGName() const override
     {
-        return QString::fromLatin1("Sketcher_Pointer_Create_Ellipse");
+        if (constructionMethod() == DrawSketchHandlerEllipse::ConstructionMethod::Center) {
+            return QString::fromLatin1("Sketcher_Pointer_Create_EllipseByCenter");
+        }
+        else if (constructionMethod() == DrawSketchHandlerEllipse::ConstructionMethod::ThreeRim) {
+            return QString::fromLatin1("Sketcher_Pointer_Create_Ellipse_3points");
+        }
+        else {

```

Comment:
Do we actually have a else case? I don't think so, so you can just do : 
```
        if (constructionMethod() == DrawSketchHandlerEllipse::ConstructionMethod::Center) {
            return QString::fromLatin1("Sketcher_Pointer_Create_EllipseByCenter");
        }
        else {
            return QString::fromLatin1("Sketcher_Pointer_Create_Ellipse_3points");
        }
```

Also shouldn't 'Sketcher_Pointer_Create_Ellipse.svg' be removed if its replaced by the 2 other icons?

Reply:
currently there is no other case, removed it